raft/node.go

```diff
@@ -22,6 +22,13 @@ import (
 	pb "github.com/coreos/etcd/raft/raftpb"
 )
 
+type SnapshotStatus int
+
+const (
+	SnapshotSuccess SnapshotStatus = 1
```

Comment:
s/SnapshotSuccess/SnapshotFinish/?
`success` probably sounds that the snapshot has been processed successfully by the remote.